packages/core/src/Account.ts

```diff
@@ -582,7 +583,7 @@ export class Account extends TypedEventEmitter<Events> {
     });
 
     const handleMissedNotifications = async (notificationId: string) => {
-      if (this.cryptoProtocolConfig?.mls && this.backendFeatures.supportsMLS) {
+      if (this.service?.mls) {
```

Comment:
It's enough to check if mls service exists - it wouldn't get initialised if mls was disabled.